//package com.sincetimes.redwar.game.comnunicate.protocol;


message RseUpdateMisc
{
	optional int32 ret = 1;
}